src/main/java/com/metamx/cache/CaffeineCache.java

```diff
@@ -156,6 +156,8 @@ public void doMonitor(ServiceEmitter emitter)
     emitter.emit(builder.build("query/cache/caffeine/total/requests", newStats.requestCount()));
     emitter.emit(builder.build("query/cache/caffeine/delta/loadTime", deltaStats.totalLoadTime()));
     emitter.emit(builder.build("query/cache/caffeine/total/loadTime", newStats.totalLoadTime()));
+    emitter.emit(builder.build("query/cache/caffeine/delta/evictionWeight", deltaStats.evictionWeight()));
```

Comment:
since we already use `sizeBytes` for the weightedSize, why not just use `evictionBytes` for eviction to make it consistent?